dashboards/grafana-dashboard-assisted-installer-resource-utilization.configmap.yaml

```diff
@@ -508,6 +508,135 @@ data:
             "align": false
           }
         },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "description": "",
+          "fieldConfig": {
+            "defaults": {
+              "links": []
+            },
+            "overrides": []
+          },
+          "fill": 0,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 24,
+            "x": 0,
+            "y": 15
+          },
+          "hiddenSeries": false,
+          "id": 37,
+          "legend": {
+            "alignAsTable": true,
+            "avg": true,
+            "current": true,
+            "max": true,
+            "min": true,
+            "rightSide": true,
+            "show": true,
+            "total": false,
+            "values": true
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "8.5.2",
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [
+            {
+              "$$hashKey": "object:129",
+              "alias": "99th percentile",
+              "yaxis": 2
+            }
+          ],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "gH4yqWBnz"
+              },
+              "editorMode": "code",
+              "expr": "avg(rate(container_cpu_usage_seconds_total {namespace=\"$namespace\",pod=~\"assisted-service-.*\"\n} [5m]))",
```

Comment:
Ideally we should use variable rate time window, we'll follow up in a later PR to provide options as `$__rate_interval` is not suitable in this case (curve not smooth enough)

Reply:
Example charts have selected 3 months view:

Chart with `$__rate_interval`:
![throttling_rate_interval](https://user-images.githubusercontent.com/18526422/177973504-2e4fa78b-b9bb-4754-945b-24965dd3748e.png)

Chart with `5m` interval hardcoded:
![throttling_5m_interval](https://user-images.githubusercontent.com/18526422/177973543-d1697c11-dd3c-4612-9b8c-f5f520159362.png)

Ideally we should add a global variable to select interval, so we are able to change it when selecting larger/small timeframes, as `$__rate_interval` is not behaving as we'd like it to

Reply:
A Grafana bug?